### ISSUE TITLE
Chart uses networking.k8s.io/v1beta1 for Ingress, possible problem with K8S 1.22+

### ISSUE THREAD

**Issue by patrickdung** [anchore/anchore-charts#185]
https://github.com/anchore/anchore-charts/blob/master/stable/anchore-engine/templates/ingress.yaml#L1-L3

It would cause problem for K8S version 1.22+
Should detect if networking.k8s.io/v1 is usable and use it instead.

**Comment by ChaunceyShannon** [comment]
Do have the problem. 

```
$ flux reconcile helmrelease anchore 
► annotating HelmRelease anchore in flux-system namespace
✔ HelmRelease annotated
◎ waiting for HelmRelease reconciliation
✗ HelmRelease reconciliation failed: Helm upgrade failed: unable to recognize "": no matches for kind "Ingress" in version "extensions/v1beta1"
```

My Kubernetes version is:

```
$ k version                                                                                                                                              1 ↵
Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.0", GitCommit:"ab69524f795c42094a6630298ff53f3c3ebab7f4", GitTreeState:"clean", BuildDate:"2021-12-07T18:16:20Z", GoVersion:"go1.17.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.1", GitCommit:"86ec240af8cbd1b60bcc4c03c20da9b98005b92e", GitTreeState:"clean", BuildDate:"2021-12-16T11:34:54Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by Btodhunter** [comment]
fixed with #186
